datalad/config.py

```diff
@@ -73,12 +79,35 @@ def get_git_version(runner=None):
                       protocol=StdOutErrCapture)['stdout'].split()[2]
 
 
-def _where_reload(obj):
+def _scope_reload(obj):
     """Helper decorator to simplify providing repetitive docstring"""
-    obj.__doc__ = obj.__doc__ % _where_reload_doc
+    obj.__doc__ = obj.__doc__ % _scope_reload_doc
     return obj
 
 
+#
+# TODO: remove when deprecated 'where' is removed
+#
+def _where_to_scope(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if 'where' in kwargs:
+            if 'scope' in kwargs:
```

Comment:
todo -- add a unittest for this deprecation logic so it is test covered and that there is no surprises when/if it is used

Reply:
done in 05a909c449aa7e3c44ed5f0b2e4c4ab7d20fc516